### DISCUSSION TITLE
Cannot read properties of undefined (reading '_x_refs')

### DISCUSSION THREAD

**Discussion by krishnahimself** [alpinejs/alpine#2041] [1. Help]
Hi,
I am using TALL stack for my project and when I added turbolinks I am getting the following error on the console and my alpine JS related functionality stopped working. I am using Alpine version 3 and using persist plugin as well. Please help
Thanks.
![Screenshot from 2021-09-10 01-33-55](https://user-images.githubusercontent.com/19906424/132754209-bb7863d2-ab46-44fb-b5c5-bc7ea1e3f746.png)

**Comment by JordivdB** [comment]
Same problem using livewire 2.6.7 + Alpine 3.4.2 + Turbo 0.7.0.0 rc2. Can't get my head around it.

**Reply by invaders-xx** [reply]
Any news I have also this issue using livewire 2.6.7 and Alpine 3.4.2
